internal/rbd/controllerserver.go

```diff
@@ -591,6 +591,17 @@ func (cs *ControllerServer) createVolumeFromSnapshot(
 
 		return err
 	}
+
+	defer func() {
+		if err != nil {
+			log.DebugLog(ctx, "Removing the clone image %q:", rbdVol)
+			errDefer := librbd.RemoveImage(rbdVol.ioctx, rbdVol.RbdImageName)
```

Comment:
same here... is there a reason to call `librbd.RemoveImage()` instead of the more complete `rbdVol.deleteImage()`?

Reply:
No specific reason, will update